### PR TITLE
Update hooks.md

### PR DESCRIPTION
**Pull request by gbudau**
Hello, thank you guys for creating this documentation, amazing work!

Hooking was a bit confusing for me because the window open and close instantly without pressing any key (minilibx Linux version).

Let me know what you think. Thank you!

### CODE REVIEW

File: libs/minilibx/hooks.md [hunk 1]
```diff
@@ -42,29 +42,30 @@ we?
...
 typedef struct  s_vars {
     void        *mlx;
     void        *win;
 }               t_vars;
 
-int             close(int keycode, t_vars *vars)
+int             key_hook(int keycode, t_vars *vars)
 {
-    mlx_destroy_window(vars->mlx, vars->win);
+	printf("Hello from key_hook!\n");
```
Comment (harm-smits) [inline libs/minilibx/hooks.md]: Please use spaces for indentation.

**Review by harm-smits — CHANGES_REQUESTED** [review]
Just change indentation. Besides that it looks good to me. Thanks!

File: libs/minilibx/hooks.md (hunk 1 above)
Comment (gbudau) [reply libs/minilibx/hooks.md]: Thank you.

**Comment by gbudau** [comment]
Fixed indentation, thank you a lot!

BTW, would it be better to rebase and squash the commits or is it ok like this?
I was thinking about it and was not sure if the review would get affected.

**Comment by harm-smits** [comment]
I can do that myself, not a big deal. Dont worry about it ;) Thanks for your help!

**Review by harm-smits — APPROVED** [review]

**Pull request merged**